spec/controllers/photos/destroy_spec.rb

```diff
@@ -0,0 +1,35 @@
+# coding: utf-8
+require 'rails_helper'
+
+RSpec.describe PhotosController, type: :controller do
+  before do
+    @photo_house = create(:photo_house)
+    @photo_mountains = create(:photo_mountains)
+    @photo_waterfall = create(:photo_waterfall)
+    @photo_way = create(:photo_way)
+
+    subject
```

Comment:
subject ですが、使ってもあまり効果がなさそうなので無理に使わなくていい気がします
```suggestion
    delete :destroy, params: { id: @photo_way.id }
```

使い方については↓などを確認しておいてください
https://qiita.com/jnchito/items/42193d066bd61c740612#subject-%E3%82%92%E4%BD%BF%E3%81%A3%E3%81%A6%E3%83%86%E3%82%B9%E3%83%88%E5%AF%BE%E8%B1%A1%E3%81%AE%E3%82%AA%E3%83%96%E3%82%B8%E3%82%A7%E3%82%AF%E3%83%88%E3%82%921%E7%AE%87%E6%89%80%E3%81%AB%E3%81%BE%E3%81%A8%E3%82%81%E3%82%8B